src/styles/theme.ts

```diff
@@ -1,19 +1,20 @@
 import { createTheme } from '@mui/material'
 
 declare module '@mui/material/styles' {
-  interface Theme {
-    status: {
-      danger: React.CSSProperties['color']
-    }
-  }
-
   interface Palette {
     neutral: Palette['primary']
   }
 
   interface PaletteOptions {
     neutral: PaletteOptions['primary']
   }
+  interface Palette {
```

Comment:
Shouldn't we just merge the `Palette` and `PaletteOptions` with the previous definitions of them contained in this file?